docs/deployment/feature_gates.md

```diff
@@ -26,6 +26,7 @@ The following tables are a summary of the feature gates that you can set on diff
 | ShootCredentialsBinding                  | `false` | `Alpha` | `1.98`  | `1.106` |
 | ShootCredentialsBinding                  | `true`  | `Beta`  | `1.107` |         |
 | NewWorkerPoolHash                        | `false` | `Alpha` | `1.98`  |         |
+| NewWorkerPoolHash                        | `true`  | `Beta`  | `1.124` |         |
```

Comment:
You need to add an entry for 1.123 in  https://github.com/gardener/gardener/blob/f707e421f160dd487c254c681f8d669ed68a40b5/pkg/features/features.go#L43